blogs/laceyhenschel.com.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "Lacey Williams Henschel's blog",
+  "type": "blog",
+  "url": "https://www.laceyhenschel.com/blog",
+  "photo_url": "https://avatars.githubusercontent.com/u/2286304?v=4",
+  "description": "Occasional posts about Python and Django",
+  "language": "en",
+  "authors": [ 
+    {
+      "name": "Lacey Williams Henschel",
+      "social_media": [{
+         "github": "williln",
+         "website": "https://www.laceyhenschel.com",
+         "linkedin": "laceynwilliams",
```

Comment:
The trailing comma should be removed here - otherwise it will throw errors when using the data :)